lib/Installation/ProductSelection/ProductSelectionPage.pm

```diff
@@ -33,8 +34,9 @@ sub init {
 }
 
 sub is_shown {
-    my ($self) = @_;
-    return $self->{rdb_SLES}->exist();
+    my ($self, %args) = @_;
+    my $timeout = 120 * get_var('TIMEOUT_SCALE', 1);
```

Comment:
this need to be done in the level of Test layer, Page layer only cares about being a repository of ids according to the pattern, that is why most of the function are really simple, doing it here breaks the design.

Reply:
OK, I'd think about a way to fix this. Thanks.